src/helpers/typeAliases.ts

```diff
@@ -0,0 +1,2 @@
+export type Primitive = string | number | bigint | boolean | null | undefined;
+export type Scalars = Primitive | Primitive[];
```

Comment:
Is `Scalars` actually use in this PR?

Reply:
Nope, and I do not actually know whether it is even used anywhere in the repo, is it?

I moved it to the typeAliases file, because it is very closely related to the Primitive type (which I needed to move). Also it was defined right under it.